src/cdk/v2/bindings/default.js

```diff
@@ -26,11 +29,18 @@ function isValidEventType(event) {
 
 function assert(val, message, status, statTags, destination) {
```

Comment:
We should remove from definition too right?

Reply:
Fixed.